changelogs/README.md

```diff
@@ -0,0 +1,5 @@
+# Changelog and Release Notes
+
+For information regarding the changelog and release notes process, please consult the README in the GitHub Actions repository that this process utilizes. To view this README, follow the link below:
+
+[GitHub Actions Workflow README](https://github.com/BigSamu/OpenSearch_Change_Set_Create_Action/blob/main/README.md)
```

Comment:
nit - prefer contextual link text:

```suggestion
For information regarding the changelog and release notes process, please consult the [README in the GitHub Actions repository](https://github.com/BigSamu/OpenSearch_Change_Set_Create_Action/blob/main/README.md) that this process utilizes.
```

Reply:
Also, I do think we should have here and in the developer guide the actual release note generation command and usage: `yarn release_note:generate`.